src/MarkdownTextInput.web.tsx

```diff
@@ -310,9 +310,8 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
         return;
       }
 
-      const hostNode = (divRef.current.firstChild as HTMLElement) ?? divRef.current;
-      const newWidth = hostNode.offsetWidth;
-      const newHeight = hostNode.offsetHeight;
+      const newWidth = divRef.current.offsetWidth;
+      const newHeight = divRef.current.offsetHeight;
```

Comment:
I would probably do it like this, but that's my personal preference:

```suggestion
      const { offestWidth, offsetHeight } = divRef.current;
```

Or like this:

```suggestion
      const { offestWidth: newWidth, offsetHeight: newHeight } = divRef.current;
```

Reply:
`offest` -> `offset`